gnocchi/tests/test_rest.py

```diff
@@ -1591,6 +1591,33 @@ def test_search_resources_with_details_via_accept(self):
                 "/v1/search/resource/generic",
                 headers={"Accept": "application/json; details=true"}))
 
+    def _do_test_list_resources_with_attrs(self, request):
```

Comment:
this would be better as gabbi test but meh.

can you also verify: ?attrs=id&attrs=user_id works. i'm more accustomed to this formatting and i think most frameworks handle this natively.

also, you should add negative tests to tests:
- if you give an attribute that does not exist.
- if you give no attrs, you get all attributes

Reply:
yes please gabbi :(